src/election.js

```diff
@@ -183,7 +183,7 @@ export default class Election {
                     this.phase = 'cancelled';
 
                     // Convert link to chat-friendly markup
-                    this.cancelledText = statusElem.html()
+                    this.cancelledText = $(statusElem).html()
```

Comment:
need to test on a cancelled election, e.g.: https://astronomy.stackexchange.com/election/1

Reply:
Definitely will - I was also thinking about doing that, but needed to find one first, thanks!